frontend/src/pages/archive/crawl-templates-new.ts

```diff
@@ -141,6 +140,17 @@ export class CrawlTemplatesNew extends LiteElement {
       this.exclusions = this.initialCrawlTemplate.config.exclude;
     }
     this.browserProfileId = this.initialCrawlTemplate.profileid;
+    // Default to current user browser language
+    const browserLanguage = navigator.languages?.length
+      ? navigator.languages[0]
+      : navigator.language;
```

Comment:
Could probably just use `navigator.language` here, since:
```
The value of [navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language) is the first element of the returned array.
```
per https://developer.mozilla.org/en-US/docs/Web/API/Navigator/languages

Reply:
Ah yeah, habit!